### DISCUSSION TITLE
Mongoose: type, validation and default value, is possible?

### DISCUSSION THREAD

**Discussion by RavenPoe1900** [Automattic/mongoose#12555] [Q&A]
Hello, I am creating a field of the user table like this..

   

```
removedId:{
     type:String,
     validate:validator.isMongoId,
     default:null,}

```

and when I try to create it gives me the following error: ERROR [ExceptionsHandler] User validation failed: removedId: Expected a string but received a null

**Comment by vkarpov15** [comment]
`removedId` is null by default, so if you don't specify `removedId` you'll get this error.

I'd also recommend doing `validate: v => validator.isMongoId(v)` rather than `validate: validator.isMongoId` to be explicit about function arity.
